src/documentation/components/ThemeSwitcher.vue

```diff
@@ -1,7 +1,7 @@
 <template>
```

Comment:
Formatting looks off, was prettier run on this file?

Reply:
The file is formatted using prettier. No issues in linting
<img width="156" alt="Screenshot 2021-11-05 at 15 15 04" src="https://user-images.githubusercontent.com/25136131/140524472-93d05bfb-b6f4-4122-b6e1-dd1cdfde913b.png">

Reply:
Ack!